File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.js
function solution(n) {
    var answer = 0;
    const str = String(n)
    const newArr = Array.from(str)
    // console.log(newArr)
    for(let i = 0; i < newArr.length; i++){
        answer += Number(newArr[i])
    }
    return answer;
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int answer = 0;
        String str = Integer.toString(n);
        String[] arr = str.split("");
        
        for(int i = 0; i < arr.length ; i++){
            System.out.println(arr[i]);
            answer += Integer.valueOf(arr[i]);
        };

        // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
        // System.out.println(arr.toString());

        return answer;
    }
}